### ISSUE TITLE
Shroom bug persists

### ISSUE THREAD

**Issue by Graydenn** [R3nzTheCodeGOD/R3nzSkin#3]
Unfortunately, even with the new base, this bug persists.

![download](https://user-images.githubusercontent.com/42272058/134753433-730b36b9-a958-46b7-9e8e-295b798721f2.png)

Exact steps to reproduce:
1. Open a Practice Tool match as Teemo with the Cottontail skin (Probably works with other skins?)
2. Level up to Level 6, put a point in your ultimate
3. Place one shroom, should be normal. Change the skin to Beemo skin, and the shroom will update correctly
4. Place another shroom after the update finishes, and it will have the Beemo shape with the Cottontail texture as pictured

**Comment by Graydenn** [comment]
I tested it a bit more and this does not happen if you start the game with the base skin instead of the Cottontail skin. Furthermore, this bug seems exclusive to starting the game with the Cottontail skin and then switching to the Beemo skin. It didn't happen if I started the game with the Spirit Blossom skin and then switched to the Beemo skin.

Guess the fix is to just not start the game with the Cottontail skin, unless it can be fixed.
